domain/src/main/kotlin/no/nav/su/se/bakover/domain/beregning/BeregningMedFradragBeregnetMånedsvis.kt

```diff
@@ -47,15 +47,22 @@ data class BeregningMedFradragBeregnetMånedsvis(
                 periode = it,
                 sats = sats,
                 fradrag = beregnetPeriodisertFradrag[it] ?: emptyList(),
-                fribeløpForEps = fradragStrategy.getEpsFribeløp(it)
+                fribeløpForEps = fradragStrategy.getEpsFribeløp(it),
             ).let { månedsberegning ->
-                when (månedsberegning.ytelseStørreEnn0MenMindreEnnToProsentAvHøySats()) {
-                    true -> MånedsberegningFactory.ny(
-                        periode = månedsberegning.periode,
-                        sats = sats,
-                        fradrag = månedsberegning.getFradrag()
-                            .plus(månedsberegning.lagFradragForBeløpUnderMinstegrense()),
+                when (
+                    størreEnn0MenMindreEnnToProsentAvHøySats(
+                        månedsberegning.sumYtelseUtenSosialstønad(),
```

Comment:
Hvordan henger dette sammen med det som foregår i `VurderAvslagGrunnetBeregning`? Er det ikke litt rart at vi filtrerer vekk fradrag i forkant der, mens vi her gjør en filtrering internt for hver måned? Det føles som dette ikke henger helt sammen?